clang/runtime/dpct-rt/include/device.hpp.inc

```diff
@@ -1060,6 +1054,26 @@ static inline sycl::queue &get_default_queue() {
 }
 // DPCT_LABEL_END
 
+// DPCT_LABEL_BEGIN|typedef_queue_ptr|dpct
+// DPCT_DEPENDENCY_BEGIN
+// Device|get_default_queue
+// DPCT_DEPENDENCY_END
+// DPCT_CODE
+
+class queue_ptr {
+  sycl::queue* _q;
+public:
+  queue_ptr(sycl::queue* _p)
+    : _q(_p ? _p : &get_default_queue()) {}
```

Comment:
dpct::get_default_queue should also return queue_ptr since we should hide sycl::queue in all migration results.

Reply:
I'm not sure I why this change should be made/the reason behind it. Why should `get_default_queue` not return a reference to a `sycl::queue`? Also, this would incur many changes, as there are many places our migration expects `get_default_queue` to a be a reference to `sycl::queue` (e.g. member access through `.` : `q_ct1.parallel_for(...)`).